lib/plotting_functions.py

```diff
@@ -80,7 +81,6 @@ def global_average(fld, wgt, verbose=False):
             break
     fld2 = np.ma.masked_invalid(fld)
     if verbose:
-        print("(global_average)-- fraction input missing: {}".format(fraction_nan(fld)))
```

Comment:
Also @brianpm was there a reason you removed this `print` statement?

Reply:
Yes, because `fraction_nan` did not seem to be defined. Maybe I missed it?